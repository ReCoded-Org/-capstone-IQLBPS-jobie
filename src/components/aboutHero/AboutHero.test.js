import renderer from "react-test-renderer";

import AboutHero from "./AboutHero";

it("renders correctly when the component matches the snapshot", () => {
  const tree = renderer.create(<AboutHero />).toJSON();
  expect(tree).toMatchSnapshot();
});
