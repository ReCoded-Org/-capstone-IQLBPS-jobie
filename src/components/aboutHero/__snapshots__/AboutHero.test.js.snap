// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when the component matches the snapshot 1`] = `
<div
  className="bg-gray-500/5 flex justify-around items-center px-32 small:px-14 small:py-5 medium:py-6 medium:px-16 large:py-8 large:px-18 pt-12 pb-10 pl-23 small:flex-col"
>
  <div
    className=" p-8 "
  >
    <h4
      className="text-dark-gray text-5xl small:text-lg medium:text-2xl large:text-3xl font-bold"
    />
    <p
      className="text-dark-gray text-xl  medium:text-yellow-30 font-inter small:text-xs medium:text-lg large:xl pt-5 small:pt-1 medium:pt-2 large:pt-4"
    />
  </div>
  <div
    className=" p-8 small:pt-3 "
  >
    <img
      alt="ChitChat Logo"
      className=" h-auto w-500 small:w-200 medium:w-350 "
    />
  </div>
</div>
`;
